flink-runtime/src/main/java/org/apache/flink/runtime/net/SSLUtils.java

```diff
@@ -285,7 +291,14 @@ private static KeyManagerFactory getKeyManagerFactory(
                                 : SecurityOptions.SSL_REST_KEY_PASSWORD,
                         SecurityOptions.SSL_KEY_PASSWORD);
 
-        KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+        // do not use getAndCheckOption here as there is no fallback option and a default is
+        // specified
+        String keystoreType =
+                internal
+                        ? config.get(SecurityOptions.SSL_INTERNAL_KEYSTORE_TYPE)
+                        : config.get(SecurityOptions.SSL_REST_KEYSTORE_TYPE);
```

Comment:
Just a clarification for other reviewers, since there is default value it just doesn't make sense to provide fallback.